apps/web/src/state/farmsV4/state/accountPositions/fetcher/v3/getAccountV3FarmingPendingCakeReward.ts

```diff
@@ -0,0 +1,49 @@
+import { ChainId } from '@pancakeswap/chains'
+import { masterChefV3ABI } from '@pancakeswap/v3-sdk'
+import { getMasterChefV3Contract } from 'utils/contractHelpers'
+import { Address, decodeFunctionResult, encodeFunctionData, Hex } from 'viem'
+
+export const getAccountV3FarmingPendingCakeReward = async (
```

Comment:
there is `useStakedPositionsByUser` existing, why don't just reuse it by pass a chainId

Reply:
actually it is a copycat of `useStakedPositionsByUser`, I dont want to modify the legacy code, which will helpful when we do code clean up for dead code